src/playlist-creator.tsx

```diff
@@ -16,6 +16,7 @@ export const PlaylistCreator: React.FunctionComponent<{ authService: AuthService
     const [user, setUser] = useState<User>();
     const [selectedGenres, setSelectedGenres] = useState<string[]>([]);
     const [selectedArtists, setSelectedArtists] = useState<string[]>([]);
+    const [recommendationLimit, setRecommendationLimit] = useState<number>(20);
```

Comment:
Why store this value both here and inside the slider component? You can pass it as a prop to the slider